server/journeys/referral/confirmation/view.njk

```diff
@@ -0,0 +1,21 @@
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Confirmation - Make a CSIP referral" %}
+
+{% set theHtml = "<div><strong>" + (prisoner.lastName + ", " + prisoner.firstName) | convertToTitleCase + "</strong></div><div>" + prisoner.prisonerNumber + "</div>" %}
```

Comment:
this can go directly into govukPanel without declaring a variable?

also we might want `| escape` to sanitise values for `html`